tests/php/Forms/DatetimeFieldTest.php

```diff
@@ -141,14 +142,14 @@ public function testSetValueWithLocalised()
 
         $datetimeField
             ->setHTML5(false)
-            ->setLocale('en_NZ');
+            ->setLocale('de_DE');
```

Comment:
We use `de_DE` in the `DateFieldTest` for localisation testing.
`en_NZ` is one of the locales with the upstream bug so we can't get a reliable value from it.